LayoutTests/fast/dom/normalize-doesnt-check-string-length.html

```diff
@@ -0,0 +1,25 @@
+<html>
+<head>
+    <script src="../../resources/js-test.js"></script>
+    <script>
+        function f1() {
+            const MAX_LENGTH = 1 << 28;
+
+            const largeString = 'A'.repeat(MAX_LENGTH);
+
+            const container = document.getElementById('container');
+
+            for (let i = 0; i < 9; ++i)
+                container.appendChild(document.createTextNode(largeString));
+
+            shouldThrowErrorName('container.normalize()', 'InvalidModificationError');
+            description("This test passes if webkit doesn't crash and there's an exception");
+        }
+
+        window.testRunner?.dumpAsText();
```

Comment:
I don't think this is needed when using js-test.js

Reply:
Yes, please remove this. Also, you can use the `make-new-script-test` script to create a test from a template.

Reply:
This is still not removed yet.